nf_core/pipeline-template/lib/NfcoreTemplate.groovy

```diff
@@ -222,6 +223,21 @@ class NfcoreTemplate {
         }
     }
 
+    //
+    // Dump pipeline parameters in a json file
+    //
+    public static void dump_parameters(params) {
+        def output_d = new File("${params.outdir}/pipeline_info/")
+        if (!output_d.exists()) {
+            output_d.mkdirs()
+        }
+
+        def output_pf  = new File(output_d, "params.json")
```

Comment:
I suspect that this will crash if the file already exists (eg. When resuming a pipeline). We should probably try to avoid this by checking if the file already exists, and/or incorporating a timestamp into the filename.

Reply:
I tested it and it did not crashed but yes, probably better to be in the safe lane, now is updated